src/contribute/release-guide.md

```diff
@@ -75,7 +75,7 @@ This will list your GPG keys. One of these should reflect your Apache account, f
 
 Here, the key ID is the 8-digit hex string in the `pub` line: `845E6689`.
 
-Now, add your Apache GPG key to the Beam’s `KEYS` file both in [`dev`](https://dist.apache.org/repos/dist/dev/beam/KEYS) and [`release`](https://dist.apache.org/repos/dist/release/beam/KEYS) repositories at `dist.apache.org`. Follow the instructions listed at the top of these files.
+Now, add your Apache GPG key to the Beam’s `KEYS` file both in [`dev`](https://dist.apache.org/repos/dist/dev/beam/KEYS) and [`release`](https://dist.apache.org/repos/dist/release/beam/KEYS) repositories at `dist.apache.org`. Follow the instructions listed at the top of these files. (Note: Not all committers have write access to the [`release`](https://dist.apache.org/repos/dist/release/beam/KEYS) repository. If you end up getting 403 errors ask a PMC member for assistance.)
```

Comment:
not all committers -> only PMC members
remove link, no need to repeat it so many times
If you end up getting a 403 error, ask on the mailing list for assistance.